bitcoin/src/sign_message.rs

```diff
@@ -1,39 +1,34 @@
 // Written in 2014 by Andrew Poelstra <apoelstra@wpsoftware.net>
 // SPDX-License-Identifier: CC0-1.0
 
-//! Miscellaneous functions.
+//! Signature
 //!
-//! This module provides various utility functions including secp256k1 signature
-//! recovery when library is used with the `secp-recovery` feature.
+//! This module provides signature related functions including secp256k1 signature recovery when
+//! library is used with the `secp-recovery` feature.
```

Comment:
In 07a8591d668aa3a6fc49bb23ad58e5256a2cfac7:

This should say "signmessage" rather than "signature" and explicitly say that it's about the old Bitcoin Core signmessage protocol. We don't need to mention pubkey recovery or anything except to say that this module requires the `secp-recovery` feature to be enabled.